sdxlib/sdx_client.py

```diff
@@ -517,7 +547,9 @@ def create_l2vpn(self) -> SDXResponse:
             return SDXResponse(response_json)
 
         try:
-            response = requests.post(url, json=payload, timeout=120)
+            # Include authentication if username and password are provided.
+            auth = (self._http_username, self._http_password)
```

Comment:
Hi @SColson82 since you are using the attribute (`self._http_username` and `self._http_password`) instead of the property (`self.http_password`), I wonder if the property is really necessary. Maybe you change all the usage of those attributes to refer to the property. I mean:
```suggestion
            auth = (self.http_username, self.http_password)
```

Reply:
Hi @SColson82 @italovalcy, excellent initiative; please take a look here: 

https://docs.google.com/document/d/1oRfxWk1q560OY-Seo5_ZVCuYWNc9zwmC4LseEHrMVuI/edit?tab=t.0

Feel free to adapt the code and documentation based on the specifications. 

Page 6: FABRIC, OpenID Connect (OIDC), and SDX-Lib

Reply:
@SColson82 can you please double check the points I mentioned before about the usage of private attributes versus getter/property function?